.github/workflows/ci.yml

```diff
@@ -2,7 +2,12 @@ name: ci
 on:
   push:
     branches: [ "main" ]
-  workflow_dispatch: {} # support manual runs
+  workflow_dispatch:
+    inputs:
+      plugins:
+        description: "Plugins to build and publish (i.e. connect-go:latest, connect-go, grpc/java:v1.53.0)"
+        default: ''
+        type: string
```

Comment:
This allows a user to override the calculated changed plugins with a manual build to provide their own inputs on which plugins should be built/tested/published.